projects/packages/my-jetpack/_inc/data/products/use-update-historically-active-modules.ts

```diff
@@ -0,0 +1,27 @@
+import {
+	QUERY_UPDATE_HISTORICALLY_ACTIVE_MODULES_KEY,
+	REST_API_UPDATE_HISTORICALLY_ACTIVE_MODULES,
+} from '../constants';
+import useSimpleMutation from '../use-simple-mutation';
+
+const useUpdateHistoricallyActiveModules = () => {
+	const { mutate: updateHistoricallyActiveModules } = useSimpleMutation< JetpackModule[] >( {
+		name: QUERY_UPDATE_HISTORICALLY_ACTIVE_MODULES_KEY,
+		query: {
+			path: REST_API_UPDATE_HISTORICALLY_ACTIVE_MODULES,
+			method: 'POST',
+		},
+		options: {
+			onSuccess: data => {
+				// Update window state with new data
+				if ( data.length > 0 ) {
+					window.myJetpackInitialState.lifecycleStats.historicallyActiveModules = data;
+				}
+			},
+		},
```

Comment:
This one is okay to fail quietly, we don't want to show an error message for a failed request for data that users really know nothing about